<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>historiaDrukDialog</class>
 <widget class="QDialog" name="historiaDrukDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1911</width>
    <height>948</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Historia wydruk√≥w</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>7</y>
     <width>1102</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="lineEditroz">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>94</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditnr">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>94</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditKlSk">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>94</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditKlnr">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>94</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditwz">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>94</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditsp">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>94</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditkol">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>94</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditoc">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>94</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditma">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>94</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditwkl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>94</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>3</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="zasoby.qrc">
     <normaloff>:/zasoby/reset.png</normaloff>:/zasoby/reset.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSzukaj">
   <property name="geometry">
    <rect>
     <x>1115</x>
     <y>3</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="zasoby.qrc">
     <normaloff>:/zasoby/find.png</normaloff>:/zasoby/find.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>1901</width>
     <height>941</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>23</number>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
